<ui version="4.0" >
 <class>TarifaBase</class>
 <widget class="QWidget" name="TarifaBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tarifa</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >
    <normaloff>:/Images/product.svg</normaloff>:/Images/product.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/BfBulmaFact48x48/images/png/48x48/i_bulmages_tarifa.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item row="0" column="0" >
         <widget class="QToolButton" name="mui_guardar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar</string>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/edit-save.svg</normaloff>:/Images/edit-save.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/edit-delete.svg</normaloff>:/Images/edit-delete.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Actualizar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/view-refresh.svg</normaloff>:/Images/view-refresh.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>399</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel12" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Nombre de la tarifa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mui_nomtarifa" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="mui_filtro" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Filtrado de articulos:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item rowspan="3" row="0" column="0" >
       <widget class="BfBuscarFamilia" native="1" name="mui_idfamilia" >
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Almacen:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>290</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="BfBuscarAlmacen" name="mui_almacen" >
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>290</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>445</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="ListLTarifaView1" name="mui_list" />
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>363</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar" >
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar" >
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <customwidgets>
  <customwidget>
   <class>ListLTarifaView1</class>
   <extends>QTableWidget</extends>
   <header>listltarifaview.h</header>
  </customwidget>
  <customwidget>
   <class>BfBuscarAlmacen</class>
   <extends>QComboBox</extends>
   <header>bfbuscaralmacen.h</header>
  </customwidget>
  <customwidget>
   <class>BfBuscarFamilia</class>
   <extends></extends>
   <header>bfbuscarfamilia.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
